src/ice_shelf/MOM_ice_shelf_state.F90

```diff
@@ -46,9 +46,13 @@ module MOM_ice_shelf_state
     tflux_shelf => NULL(), &   !< The downward diffusive heat flux in the ice
                                !! shelf at the ice-ocean interface [Q R Z T-1 ~> W m-2].
 
-    tfreeze => NULL()          !< The freezing point potential temperature
+    tfreeze => NULL(), &       !< The freezing point potential temperature
                                !! at the ice-ocean interface [C ~> degC].
 
+    !only active when calve_ice_shelf_bergs=true:
+    calving => NULL(), &       !< The mass flux per unit area of the ice shelf to convert to
+                               !!bergs [RZ_T ~> kg m-2 s-1].
```

Comment:
The standard syntax for these units should be `!! bergs [R Z T-1 ~> kg m-2 s-1].`